semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -102,12 +102,29 @@ public static boolean isRpcTypeGrpcForOTelFormat(Event event) {
             event, getAttributeKeysForRpcService()));
   }
 
+  public static Optional<String> getRpcMethod(Event event) {
+    return Optional.ofNullable(
+        SpanAttributeUtils.getFirstAvailableStringAttribute(
+            event, List.of(OTEL_SPAN_TAG_RPC_METHOD.getValue())));
+  }
+
   public static Optional<String> getRpcOperation(Event event) {
     return Optional.ofNullable(
         SpanAttributeUtils.getFirstAvailableStringAttribute(
             event, RpcSemanticConventionUtils.getAttributeKeysForGrpcMethod()));
   }
 
+  public static Boolean checkIfRpcSystemIsGrpc(Map<String, AttributeValue> avm) {
```

Comment:
What do you think about name `checkIfRpcSystemIsGrpc` -> `isRpcSystemGrpc` / `isGrpcRequest`?

Reply:
Can we also rename `avm` -> `attributeValueMap` ?

Reply:
A method by the name of isRpcSystemGrpc already exists in some other file which I modified to this method. So should I rename this check method to that only ?

Reply:
So that method no longer exists? if yes, then yeah use `isRpcSystemGrpc`

Reply:
That method is in rpcfieldgenerator file

Reply:
I think we can still go ahead with the renaming